modules/audio/audio_component.cc

```diff
@@ -48,7 +47,10 @@ bool AudioComponent::Init() {
 
 bool AudioComponent::Proc(const std::shared_ptr<AudioData>& audio_data) {
   audio_info_.Insert(audio_data);
-  // TODO(all) get data for following online processings
+  AINFO << "Getting direction...";
```

Comment:
Please change it to ADEBUG for debug information

Reply:
Fixed

Reply:
I change it to `AINFO << "Current direction is: << ...`. I think it's a kind of information that's needed at least for now? (Maybe delete it before release?)